Sources/Proxy/Representations/SliceRepresentationProxy/index.js

```diff
@@ -20,14 +20,14 @@ function mean(...array) {
 
 function updateDomains(dataset, dataArray, { slicingMode }, updateProp) {
   const dataRange = dataArray.getRange();
-  const extent = dataset.getExtent();
+  const bounds = dataset.getBounds();
   const axisIndex = 'XYZ'.indexOf(slicingMode);
 
   const propToUpdate = {
     sliceIndex: {
       domain: {
-        min: extent[axisIndex * 2],
-        max: extent[axisIndex * 2 + 1],
+        min: bounds[axisIndex * 2],
+        max: bounds[axisIndex * 2 + 1],
         step: 1,
```

Comment:
Should this be updated as well to match the correct axis spacing?

Reply:
done